examples/objectdetection/src/Photo.tsx

```diff
@@ -2,13 +2,24 @@ import type { BottomTabScreenProps } from "@react-navigation/bottom-tabs";
 import React from "react";
 import { View, Text } from "react-native";
 import type { RootTabParamList } from "./navigation";
+import ImagePicker from 'react-native-image-crop-picker';
 
 type Props = BottomTabScreenProps<RootTabParamList, "Photo">;
 
 export const Photo: React.FC<Props> = () => {
   return (
     <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
       <Text>Still Photo Placeholder</Text>
+
+      
     </View>
   );
 };
+
+ImagePicker.openPicker({
+  width: 300,
+  height: 400,
+  cropping: true
+}).then(image => {
+  console.log(image);
```

Comment:
assuming that when you run the example app locally, when you select a file, you see output in the metro bundler window to the effect that this line of code executed, then this is good.